packages/hub/src/app/users/[username]/models/[slug]/page.tsx

```diff
@@ -1,13 +1,9 @@
 "use client";
 
-import { useFragment } from "react-relay";
+import { useModelPageQuery } from "./ModelPage";
+import { EditModelPageBody } from "./EditModelPageBody";
 
-import { ModelPage$key } from "@/__generated__/ModelPage.graphql";
-import { ModelPageFragment, useModelPageQuery } from "./ModelPage";
-import { ViewModelRevision } from "./ViewModelRevision";
-import { ViewModelRevisionContent } from "./ViewModelRevisionContent";
-
-export default function OuterModelPage({
+export default function Page({
```

Comment:
Yep, maybe we should give up and call all page components `Page`. It's not important (only affects React Dev Tools), and I'd prefer to avoid the cognitive load of picking the name.

Reply:
I think the layout stuff could use some rethinking. It's difficult to get that right, and we have some annoying exceptions, like the Model editor page.